index.ts

```diff
@@ -29,28 +30,32 @@ server.use(restify.gzipResponse());
 server.use(restify.fullResponse());
 server.use(restify.throttle(conf.get('throttleOptions')));
 server.use(restify.requestLogger());
-server.use(restify.queryParser());
+server.use(restify.queryParser({ mapParams: false }));
+server.use(validator());
 server.use(util.getCert);
 server.use(util.log);
 server.use(blpSession.getSession);
 server.use(requestHandler.elevateRequest);
 
-// Route
-server.post('/request/:ns/:svName/:reqName',
+// Routing
+
+// `/request` handles request/response services. There is one operation on these services, namely
+// "get me some data". In some sense this ought to be a GET but because the request data is fairly
+// large we use a POST.
```

Comment:
I would be OK if this was a `GET` instead of a `POST`.  `POST` is really intended for indicating that the state of the server is mutated, where `GET` does not imply so.  Since it is atypical for data for `GET` requests, it is not forbidden.